recipes/gpaw/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set build = 0 %}
+{% set name = "gpaw" %}
+{% set version = "1.5.1" %}
+{% set sha256 = "63a00bd97cc4e10fb5f5c718039af18c90c7e0e19897a6a852e5f54fc0c81776" %}
+{% set build_string = "py{}_{}_{}".format(environ.get('CONDA_PY', ''), mpi, build) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 001_config-py.patch
```

Comment:
Please, could you write an explanation why do you need this patch?

Reply:
Sure, should I add it as a comment in `meta.yaml`? (the explanation is that gpaw's config.py assumes libpython is a dynamic library, whereas on conda-forge it seems to be a static library).

Reply:
I've opened a merge request upstream: https://gitlab.com/gpaw/gpaw/merge_requests/525